api/app/signals/apps/signals/models.py

```diff
@@ -179,6 +180,22 @@ def update_priority(self, data, signal):
 
         return priority
 
+    def create_note(self, data, signal):
+        """Create a new `Note` object for a given `Signal` object.
+
+        :param  data: deserialized data dict
```

Comment:
Typo, double space between `param` and `data` ;)

Reply:
fixed